### PR TITLE
Release 1.11.0 snapshot

### PR DESCRIPTION
**Pull request by labkey-susanh**
**Changes**
* Removed RPackages plugin in favor of defining relevant tasks in RPackages' own `build.gradle` file
* Check for existence of `node_modules` directory before attempting to remove it with `cleanNodeModules` to avoid crankiness on Windows
* Restore input and output declarations for npmInstall as a possible fix for [Issue 40153](https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=40153)
* Remove remoteapi/java as a base module
* Change default npmInstall command to `ci` instead of `install` with a property `npmInstallCommand` that can be used to override this default (e.g., `PnpmInstallCommand=install`)

### CODE REVIEW

**Review by cnathe — COMMENTED** [review]
These changes look good to me, but I will defer to @labkey-tchad for approval / final review.

File: README.md [hunk 1]
```diff
@@ -9,8 +9,19 @@ but also make certain assumptions that you may not want to impose on your module
 
 ## Release Notes
 
-### version TBD
+## version TBD
 *Released*: TBD
+(Earliest compatible LabKey version: 20.4)
+* Removed RPackages plugin in favor of defining relevant tasks in RPackages' own `build.gradle` file
+* Check for existence of `node_modules` directory before attempting to remove it with `cleanNodeModules` to avoid crankiness on Windows
```
Comment (labkey-adam) [inline README.md:16]: :+1:

File: README.md [hunk 2]
```diff
@@ -9,8 +9,19 @@ but also make certain assumptions that you may not want to impose on your module
...
+* Restore input and output declarations for npmInstall as a possible fix for [Issue 40153](https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=40153)
+* Remove remoteapi/java as a base module
+* Change default npmInstall command to `ci` instead of `install` with a property `npmInstallCommand`
+that can be used to override this default (e.g., `PnpmInstallCommand=install`)
+* Add utility methods for getting path to sas and jdbc api projects
+
+### version 1.10.4
```
Comment (labkey-adam) [inline README.md:24]:
```suggestion
## version 1.10.4
```

File: src/main/groovy/org/labkey/gradle/task/ModuleDistribution.groovy [hunk 3]
```diff
@@ -406,6 +411,13 @@ class ModuleDistribution extends DefaultTask
                     include(name: "*")
                 }
 
+                // WEB-INF/lib (*.jar)
+                zipfileset(dir: moduleDir,
+                        prefix: "WEB-INF",
+                        erroronmissingdir: false) {
+                    include(name: "lib/*.jar")
+                }
+
```
Comment (labkey-adam) [inline src/main/groovy/org/labkey/gradle/task/ModuleDistribution.groovy:420]: Is this needed for the WAR file build?

**Review by labkey-adam — APPROVED** [review]

File: src/main/groovy/org/labkey/gradle/task/ModuleDistribution.groovy (hunk 3 above)
Comment (labkey-susanh) [reply src/main/groovy/org/labkey/gradle/task/ModuleDistribution.groovy:420]: Yes, the jar files are expected to be in the WEB-INF/lib directory.

**Pull request merged**